### ISSUE TITLE
refactor categoryid and setString

### ISSUE THREAD

**Issue by bryceAM** [bryceAM/capstone-polished#135]
db/models/products.cjs
```
...
const setString = Object.keys(fields)
        // removed the double quotes around the key, so postgres will treat the key names more forgivingly
        // .map((key, index) => `"${key}"=$${index + 1}`)
        .map((key, index) => `${key}=$${index + 1}`)
        .join(',');
...
```
api/products.cjs
```
...
const validUpdates = Object.entries(req.body).filter(([key, value]) => (key === 'name' || key === 'description' || key === 'price' || key === 'categoryId') && value);
...
```

**Comment by bryceAM** [comment]
original setString .map:
```
.map((key, index) => `"${key}"=$${index + 1}`)
```

new setString .map:
```
.map((key, index) => `${key}=$${index + 1}`)
```

**Comment by bryceAM** [comment]
Basically I changed the categoryid key to categoryId to conform to my key name convention. But my setString, which was directly passed as a database query, had to also make sure that categoryid was the column name that was used in the query, because the table column name is categoryid instead of categoryId. So I removed the double quotes in the setString around the key names in order to make postgres more forgiving of my query which now passes categoryId instead of "categoryId" into the query, thus avoiding breaking changes.
